checks/validation_compare_files.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import json
+import numpy as np
+
+from validation_createJSON import parse_resultwriter_file, parse_log_file
+
+def compare_data(new_data, validation_data):
+    '''
+    Compares new and validation data sets using numpy's isclose with a specified tolerance
+    '''
+    differences = []
+    for i, (new_entry, validation_entry) in enumerate(zip(new_data, validation_data)):
+        entry_differences = {}
+        for key in new_entry.keys():
+            if key in validation_entry:
+                if not np.isclose(new_entry[key], validation_entry[key], atol=abstolerance, rtol=reltolerance):
+                    entry_differences[key] = {
+                        'presentRun': new_entry[key],
+                        'reference': validation_entry[key]
+                    }
+        if entry_differences:
+            differences.append({'index': i, 'differences': entry_differences})
```

Comment:
```suggestion
            differences.append({'Simstep': i, 'differences': entry_differences})
```
I *think* this should be changed because the output was a bit confusing.